setup.py

```diff
@@ -38,11 +38,11 @@
         'Framework :: Django',
         'Framework :: Wagtail',
         'Framework :: Wagtail :: 2',
+        'Framework :: Wagtail :: 4',
         'Operating System :: Unix',
         'Programming Language :: Python :: 3',
-        'Programming Language :: Python :: 3.6',
-        'Programming Language :: Python :: 3.7',
```

Comment:
given this, should we mandata python>=3.8 in install requires?

Reply:
We don't currently mandate any Python version for wagtail-bakery. Python 3.7 should still work as long as you use a compatible Django.

I'm not entirely convinced mandating a Python version is strictly necessary, one should be able to use Python 3.7 (or even 3.6) with, say, Django 3.2 and Wagtail 2.15. Adding `python_requires='>=3.8'` would prevent one from doing this.

To me, the meaning of removing these versions from the classifiers means 'we don't guarantee support for those versions anymore, but feel free to try.'

My position is debatable of course. Happy to hear other insights on this subject.

Reply:
Works for me